docs/lake/design.md

```diff
@@ -559,7 +563,7 @@ values are the partial-result aggregation given by the Zed expression.
 > (except for optimizations that suit the interactive app) but rather is a powerful
 > capability for application-specific workflows that know the pre-computed
 > aggregations that they will use ahead of time, e.g., beacon analysis
-> of zeek logs.
+> of network security logs.
```

Comment:
Keeping with the theme of the prior change where I dropped the reference to `_path`, I figure there's no harm in keeping this abstract. I expect our existing Zeek-centric users will likely pick up on this more generic wording as a reference to things like RITA/similar (and if we ever wrote a blog post to that effect, that would surely drive the point home) whereas newer non-security readers could just shrug at this as less relevant to them rather than wondering "Wait, what's Zeek?"